solr/solrj/src/java/org/apache/solr/common/cloud/SolrZkClient.java

```diff
@@ -83,6 +87,45 @@ public class SolrZkClient implements Closeable {
 
   private ZkCmdExecutor zkCmdExecutor;
 
+  private final ZkMetrics metrics = new ZkMetrics();
+
+  public static class ZkMetrics implements ReflectMapWriter {
+    @JsonProperty
+    public final AtomicLong watchesFired = new AtomicLong();
```

Comment:
I think all these AtomicLongs could be replaced with LongAdder?

Reply:
I could. our serializers were not supporting `LongAdder` . I have added support for `LongAdder`